master/internal/api_experiment.go

```diff
@@ -847,6 +855,18 @@ func (a *apiServer) UnarchiveExperiment(
 	}
 }
 
+func maskContains(mask []string, v string) bool {
+	if len(mask) == 0 {
+		return true
+	}
+	for _, s := range mask {
```

Comment:
in `api_experiment.go` we've been able to use `slices.Contains(array, val)` which covers this for loop